etc/kayobe/kolla/config/nova/nova-compute-ironic.conf

```diff
@@ -0,0 +1,8 @@
+[DEFAULT]
+# Don't limit the number of concurrent builds for the nova ironic compute
+# service.
+max_concurrent_builds = 0
```

Comment:
Risky to introduce here before we forget it and scale up?

Reply:
Perhaps. I don't really have a feel for what a sensible number would be for a baremetal compute host. Certainly a lot higher than the default of 10.